ColorDeductoActivity.py

```diff
@@ -343,7 +340,7 @@ def _message_cb(self, collab, buddy, msg):
         self._processing_methods[command][0](payload)
 
     def _send_new_game(self):
-        ''' Send a new game message to all players (only sharer sends grids) '''
+        ''' Send a new game message to all players '''
```

Comment:
Your commit message says something, why remove the doc string that reiterates that?